src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -60,7 +66,7 @@ struct Partition {
   // Bit vector to store NULL flags - not using uninitialized_vector because it is not specialized for bool.
   // It is stored independently of the elements as adding a single bit to PartitionedElement would cause memory waste
   // due to padding.
-  std::vector<bool> null_values;
+  pmr_vector<bool> null_values = pmr_vector<bool>(alloc<bool>("partition_step_null_values_bitmap"));
```

Comment:
can we use auto to stick here?

Reply:
Unfortunately not, because it's a non-static struct member.